observatory-platform/observatory/platform/utils/template_utils.py

```diff
@@ -46,6 +40,8 @@
                                                  upload_files_to_cloud_storage)
 from observatory.platform.utils.jinja2_utils import make_sql_jinja2_filename, render_template
 
+partition_field = 'partition_date'
```

Comment:
This should be a parameter in each function rather than a constant, as you may want to change what field is the partition for a specific table.